dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties

```diff
@@ -38,5 +38,5 @@ mailqueue.view.updateBrowseStartPace=1000
 # Metrics get exported periodically as configured in elasticsearch.properties, thus getSize is also called periodically
 # Choose to disable it when the mail queue size is getting too big
 # Note that this is as well a temporary workaround until we get 'getSize' method better optimized
-# Optional, default true
+# Optional, default false
 mailqueue.size.metricsEnabled=true
```

Comment:
Why letting it to true in the conf file then if the default is supposed to be false?